Makefile

```diff
@@ -112,7 +112,7 @@ v8:
 	$(MAKE) -C deps/v8 $(V8_ARCH) $(V8_BUILD_OPTIONS)
 
 test: | cctest  # Depends on 'all'.
-	$(PYTHON) tools/test.py --mode=release message parallel sequential -J
+	$(PYTHON) tools/test.py --mode=release doctool message parallel sequential -J
```

Comment:
Actually, I'd keep both lists in alphabetical order if it's all the same to everyone else. 

I promise to not say another words about this, as I bike-shedded more than my share on the last topic.